kusto/ingest/ingest.go

```diff
@@ -466,7 +547,7 @@ func (i *Ingestion) Stream(ctx context.Context, payload []byte, format DataForma
 	if err := zw.Close(); err != nil {
 		return errors.E(errors.OpIngestStream, errors.KClientArgs, err).SetNoRetry()
 	}
-	if buf.Len() > 4*mib {
+	if buf.Len() > MaxStreamingSize {
```

Comment:
Mind this is a breaking change

Reply:
How? MaxStreamingSize is defined to be 4*mib

Reply:
You change the behavior of an existing API.
Someone streaming 5MB today would fail

Reply:
The if was already there, all this change is giving the constant a name.

Reply:
LOL :D 
I completely ignored this somehow